bison: 3.7.6 -> 3.8.2

trofi opened the pull request.

trofi committed:
bison: 3.7.6 -> 3.8.2

trofi committed:
bison: enable tests, enable parallelism

risicle commented:
So far have built direct bison dependendents `gecode`, `thrift`, `mono`, `flasm`, `sdcc`, `glslang`, `php74`, `php80`, `tmux` on with this on macos 10.15 and nixos x86_64. Also `bisoncpp` just on nixos. Still building more...

trofi committed:
pkgsMusl.bison: fix tests by adding iconv workaround for musl

gnulib relies on --host= to check if iconv() on the platform
might convert unsupported symbols to '*'. Noticed as failed tests.

risicle commented:
Rebuildin'...
